### PR TITLE
Adding New Blog Post and Image 

### PR DESCRIPTION
**Pull request by Bitc0indad**
new file:   public/images/Bitcoin_Lightbulb_Effect.jpg

new file:   src/content/post/Transitioning_Careers_Achieving _a_Passion_for_Code_and_Becoming_a_High-Performing_PlebNet_Developer.md

### CODE REVIEW

File: src/content/post/Transitioning_Careers_Achieving _a_Passion_for_Code_and_Becoming_a_High-Performing_PlebNet_Developer.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+## Conclusion
+Transitioning careers to become a high-performing plebnet developer with a focus on DevOps and managing tech firms requires time, dedication, and continuous learning. While the timeline provided is a rough estimate, individual progress may vary. Consistency, passion, and a continuous learning mindset are key to accelerating the learning process and achieving success. Remember to leverage internships, personal projects, and open-source contributions to gain practical experience and enhance your learning journey. By following this path and continuously expanding your skills, you can achieve your goal of becoming a highly sought-after full-stack developer in the dynamic world of technology.
+
+**Join us if you're interested in contributing as a senior developer or if you're new and want to learn basic prrogramming skills to start by [Clicking Here](https://plebnet.dev)**
```
Comment (bitkarrot) [inline src/content/post/Transitioning_Careers_Achieving _a_Passion_for_Code_and_Becoming_a_High-Performing_PlebNet_Developer.md:79]: typo, prrrograming

File: src/content/post/Transitioning_Careers_Achieving _a_Passion_for_Code_and_Becoming_a_High-Performing_PlebNet_Developer.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+**Attribution:** This content is a PlebNet Dev Contribution. The article was authored by bitcoinDad
```
Comment (bitkarrot) [inline src/content/post/Transitioning_Careers_Achieving _a_Passion_for_Code_and_Becoming_a_High-Performing_PlebNet_Developer.md:81]: make a link to your github repo at bitcoindad :)

**Review by bitkarrot — CHANGES_REQUESTED** [review]
see comments, thanks!

**Comment by bitkarrot** [comment]
Please deploy your branch to vercel.com for review, thanks!

File: src/content/post/Transitioning_Careers_Achieving _a_Passion_for_Code_and_Becoming_a_High-Performing_PlebNet_Developer.md (hunk 2 above)
Comment (Bitc0indad) [reply src/content/post/Transitioning_Careers_Achieving _a_Passion_for_Code_and_Becoming_a_High-Performing_PlebNet_Developer.md]: Added link, thank you!

File: src/content/post/Transitioning_Careers_Achieving _a_Passion_for_Code_and_Becoming_a_High-Performing_PlebNet_Developer.md (hunk 1 above)
Comment (Bitc0indad) [reply src/content/post/Transitioning_Careers_Achieving _a_Passion_for_Code_and_Becoming_a_High-Performing_PlebNet_Developer.md]: Type fixed, thanks!

**Comment by Bitc0indad** [comment]
> Please deploy your branch to vercel.com for review, thanks!

Deployed please confirm if executed correctly: 

https://vercel.com/bitc0indad/website

**Comment by plebnetdev** [comment]
> > Please deploy your branch to vercel.com for review, thanks!
> 
> Deployed please confirm if executed correctly:
> 
> https://vercel.com/bitc0indad/website

Nein. The above link does not work. it should be something like https://xsdfsdf.vercel.app

**Comment by Bitc0indad** [comment]
> > > Please deploy your branch to vercel.com for review, thanks!
> > 
> > 
> > Deployed please confirm if executed correctly:
> > https://vercel.com/bitc0indad/website
> 
> Nein. The above link does not work. it should be something like https://xsdfsdf.vercel.app

>>Thanks corrected see preview here https://website-bitc0indad.vercel.app

**Comment by plebnetdev** [comment]
Ok great. Thanks for posting the deploy link, now we can see visual issues that might not otherwise have come up from viewing the markdown only. 

For example, the title is a bit long, its also missing a ":" after "Transitioning Careers"

I would suggest you make "Achieving a Passion for Code and Becoming a High-Performing Full-Stack Developer" the subtitle and the main title "Transitioning Careers, a journey into code" or something short, no longer than 2 lines, not 4 lines on list page

Also your titles are inconsistent, please pick one, either "Achieving a Passion for Code and Becoming a High-Performing Full-Stack Developer"  or Transitioning Careers: Achieving a Passion for Code and Becoming a High-Performing PlebNet Developer" 

**Remember the first impressions count. They might be the only thing someone sees, and their memory of you as a writer**. Most people don't even read the content body, so make the title count. 


Screenshot 2023-07-12 at 12 33 05 PM

Main Blog Page: Title too long and blocky. Make a title, and then a subtitle
Screenshot 2023-07-12 at 12 38 08 PM

**Review by plebnetdev — APPROVED** [review]

**Comment by plebnetdev** [comment]
@eriknewland suggestions, please?

**Comment by eriknewland** [comment]
"Re-program your life by learning to code"
"While Living, Continue Learning: The Lifelong Loop of Knowledge Acquisition"
"Mastering Full-Stack Development: A Career Transition Fueled by Passion for Code"

**Pull request merged**